lib/OpenQA/Git.pm

```diff
@@ -81,6 +81,22 @@ sub set_to_latest_master ($self, $args = undef) {
         return $self->_format_git_error($res, 'Unable to reset repository to origin/master') unless $res->{status};
     }
 
+=comment
```

Comment:
That's usually not how pod looks like.
We can decide to document the whole function, but a lonely `=comment` directive somewhere is very uncommon. Also a `=comment` directive does not exist, as far as I know.
There is `=for comment`, but that's actually for being **not** displayed when using perldoc.

also a pod directive should have an empty line before and after.

Does `perldoc lib/OpenQA/Git.pm` show something with this change?

Reply:
https://perldoc.perl.org/perlpod